### PR TITLE
Amarell dev

### PR DESCRIPTION
**Pull request by amarell**
- Added a route for updating user
 - added get user route

### CODE REVIEW

**Comment by amarell** [comment]
@adnan-mujagic

File: controllers/userController.js [hunk 1]
```diff
@@ -65,3 +65,64 @@ module.exports.login = async (req, res) => {
...
+  if (!req.headers.authorization) {
+    return res.status(403).json({
+      message: "Unauthorized",
+    });
+  }
+
+  const loggedInUser = jwt.verify(req.headers.authorization);
+
+  if (req.params.user_id !== loggedInUser.uid) {
+    return res.status(200).json({
```
Comment (adnan-mujagic) [inline controllers/userController.js]: This should probably also return a `403` status code - Unauthorized

Comment (adnan-mujagic) [inline controllers/userController.js]:
```suggestion
    return res.status(403).json({
```

File: controllers/userController.js [hunk 2]
```diff
@@ -65,3 +65,64 @@ module.exports.login = async (req, res) => {
...
+      message: "You can only edit your own profile",
```
Comment (adnan-mujagic) [inline controllers/userController.js]:
```suggestion
      message: "Unauthorized",
```

**Review by adnan-mujagic — APPROVED** [review]

**Pull request merged**